index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Ali Kimbrell | Software Developer</title>
+		<meta name="description" content="">
+		<meta name="author" content="Ali Kimbrell">
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="/bower_components/bootstrap/dist/css/bootstrap.min.css">
+		<link rel="stylesheet" href="css/quiz.css">
+	</head>
+
+	<body>
+		<div id="wrapper">
+		<div id="background">
+			<header>
+				<!-- NAVBAR -->
+				<nav class="navbar navbar-default">
+				  <div class="myMenu container-fluid">
+				    <!-- Brand and toggle get grouped for better mobile display -->
+				    <div class="navbar-header">
+				      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+				        <span class="sr-only">Toggle navigation</span>
+				        <span class="icon-bar"></span>
+				        <span class="icon-bar"></span>
+				        <span class="icon-bar"></span>
+				      </button>
+				      <!-- <a class="navbar-brand" href="#">Brand</a> -->
+				    </div>
+				    <!-- Collect the nav links, forms, and other content for toggling -->
+				    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+				      <ul class="nav navbar-nav navbar-right">
+				        <li><a href="#">Technologies</a></li>
+				        <li><a href="#">Portfolio</a></li>
+				        <li><a href="#">About</a></li>
+				        <li><a href="#">Contact</a></li>
+				      </ul>
+				    </div><!-- /.navbar-collapse -->
+				  </div><!-- /.container-fluid -->
+				</nav>
+			</header>
+			
+				<div id="main-header">
+					<h1>Ali Kimbrell</h1>
+					<h3>Software Developer</h3>
+				</div>
+
+				<article class="block" id="technologies">
+					<h4>Technologies</h4>
+					<img src="images/bootstrap.png" class="tech">
+					<img src="images/git.png" class="tech">
+					<img src="images/github.png" class="tech">
+					<img src="images/js.jpg" class="tech">
+					<h5>Also familiar with:</h5>
+				</article>					
+
+				<article class="hidden block" id="portfolio">
+					<h4>Portfolio</h4>
+					<section>
+						PROJECT HERE
+					</section>
+					<section>
+						PROJECT HERE
+					</section>
+				</article>
+
+				<article class="block" id="about">
+					<h4>About</h4>
+					<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+					tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+					quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+					consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+					cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+					proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+				</article>
```

Comment:
I'd like to see you take the time to craft an actual bio, no matter how simple. Time spent on these details now will save you a lot of worry when you're trying to get this project deployed as a marketing tool before graduation